module/VuFindTheme/src/VuFindTheme/ThemeInfo.php

```diff
@@ -378,4 +378,27 @@ public function findInThemes($relativePathPattern)
         }
         return array_values($results);
     }
+
+    /**
+     * Search vendor directory for a file.
+     *
+     * @param string $filePath Absolute path to file
+     * @param string $fileType Specified file extension
+     *
+     * @return array|null
+     */
+    public function findInPackage($filePath, $fileType = 'js')
+    {
+        if (strpos($filePath, 'vendor/') !== false
```

Comment:
You mentioned that it might make sense to get rid of the vendor check. I guess the issue here is that if something is already web accessible through VuFind's default Apache configuration, we don't want to serve it more slowly by doing extra work. I'd have to refresh my memory a bit more to understand what checks we might be able to use to make the decision, but perhaps as a starting point, we could abstract the vendor check to a support method like `isAlreadyWebAccessible()`, which would explain our intent better and offer an opportunity to refine the logic in future. (And, of course, if I'm misremembering the purpose of the check, maybe a different name would be better).